src/common/steps/develop/developSlice.tsx

```diff
@@ -21,7 +22,7 @@ interface State {
 }
 
 const initialState: State = {
-    developState: DevelopState.CHOOSE,
+    developState: DevelopState.PREPARE_FOR_MATTER_DEV,
```

Comment:
What is the purpose of this state tracing? I can't find any information printed what is the state of the particular sample. Do we even need it?

Reply:
We need to handle these states properly